CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This file is used to list changes made in each version of the aws-parallelcluste
 - Make `aws-parallelcluster-node` daemons handle only ParallelCluster-managed Slurm partitions.
 
 **BUG FIXES**
+- Fix an issue that was causing misalignment of compute nodes EC2 DNS name on instances with multiple network interfaces.
```

Comment:
we can add a note saying that this causes issues only when using UseEc2Hostname set to true.

Reply:
tnx, added